source/includes/extracts-http-action.yaml

```
@@ -92,12 +92,20 @@ content: |
 
      * - :guilabel:`Request Body`
 
-         | ``body: <string>``
+         | ``body: <object> or <string>``
 
-       - Required. The stringified body of the HTTP request.
+       - Required. The object or stringified body of the HTTP request.
          If the request payload has a content type of
          ``multipart/form-data``, use the ``form`` parameter instead of
-         ``body``.
+         ``body``. If the request body is an object, the 
+         ``encodeBodyAsJSON`` parameter must be ``true``.
+    
+     * - :guilabel:`Encode Body as JSON`
+
+         | ``encodeBodyAsJSON: <boolean>``
+
+       - If ``true``, the body is encoded to EJSON. Only use when the ``body`` 
```

```suggestion
       - If ``true``, the body is automatically encoded as an EJSON string using ``EJSON.stringify()``. Only use when the ``body`` 
```

I wonder if we can clarify a bit more like ^